testing/consumer-test/run.ts

```diff
@@ -13,20 +13,12 @@ const versions = [/*'3.1',*/ '3.2', '3.3', '3.4', '3.5', '3.6', '3.7', '3.8', '3
 
         await pmap(
             versions,
-            async (version) => {
-                try {
-                    await execp(`npx -p typescript@${version} tsc -p tsconfig-test.json`);
-                } catch (error) {
-                    throw new Error(`Copmilation failed for typescript@${version}`);
-                }
-
-                try {
-                    await execp(`npx -p typescript@${version} tsc -p tsconfig-test.json --lib es2018`);
-                } catch (error) {
-                    throw new Error(`Copmilation failed for typescript@${version} --lib es2018`);
-                }
-            },
-            { concurrency: 3 }
+            (version) =>
+                Promise.all([
+                    execp(`npx -p typescript@${version} tsc -p tsconfig-test.json`),
```

Comment:
This actually emits better error messages that include the full command executed.

Reply:
Perfect!